pkg/kubelet/kubelet.go

```diff
@@ -466,10 +464,8 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 	httpClient := &http.Client{}
 	parsedNodeIP := net.ParseIP(nodeIP)
-	protocol := utilipt.ProtocolIPv4
 	if utilnet.IsIPv6(parsedNodeIP) {
 		klog.V(0).Infof("IPv6 node IP (%s), assume IPv6 operation", nodeIP)
```

Comment:
nit, doubt, ...
this is not really true , right? it will depend on the dual stack flag the operation mode

Reply:
single-stack IPv6 is not feature-gated

Reply:
sorry, I added the comment without context, it maybe dualstack or it may be ipv6 only at this point

Reply:
ah. Yeah, and the comment isn't really very useful... "You explicitly passed an IPv6 `--node-ip`, so just FYI, I'm going to assume you want to use IPv6".